PageObjectPattern/src/test/resources/features/apitest/spotifyGetMe.feature

```diff
@@ -0,0 +1,15 @@
+#Token access is inserted manually in SpotifyApiStep
+Feature: Sending Get requests to /me endpoint
```

Comment:
What about "Sending Get requests to /me endpoint" -> "Integration tests for v1/me endpoint"?